modules/getting-started/ref-audit-log-catalog-events.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+// assembly-audit-log.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-audit-log-catalog-events.adoc_{context}"]
+= Catalog events
+
+{product-short} audit logs can include the following catalog events:
+
+`CatalogEntityAncestryFetch`:: Tracks `GET` requests to the `/entities/by-name/:kind/:namespace/:name/ancestry` endpoint that return the ancestry of an entity
```

Comment:
@Zaperex it's the request that is returning the info (not the endpoint), right?

Reply:
It's actually the response from the endpoint that returns the information.

Reply:
Ah, ok. I've updated the language to reflect that.